diff --git a/system_psoc6.h b/system_psoc6.h
index a864f14..44759c8 100644
--- a/system_psoc6.h
+++ b/system_psoc6.h
@@ -554,7 +554,7 @@ extern "C" {
 *        <i>(USER SETTING)</i>
 *******************************************************************************/
 #if !defined (CY_CORTEX_M4_APPL_ADDR)
-    #define CY_CORTEX_M4_APPL_ADDR          (CY_FLASH_BASE + 0x2000U)   /* <<< 8 kB of flash is reserved for the Cortex-M0+ application */
+    #define CY_CORTEX_M4_APPL_ADDR          (CY_FLASH_BASE + 0x20000U)   /* <<< 128 kB of flash is reserved for the Cortex-M0+ application */
 #endif /* (CY_CORTEX_M4_APPL_ADDR) */
 
 
